misc/language-tour-guide/src/main.tsx

```diff
@@ -0,0 +1,14 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import { BrowserRouter } from "react-router";
+
+import "./styles/index.css";
+import AppRoutes from "./AppRoutes.tsx";
+
+createRoot(document.getElementById("root")!).render(
```

Comment:
Can we go with 
```js
const container = document.getElementById("root"); // or root or whatever
if(container === null) {
  throw new Error("Root not found");
}
```
instead of TS non-null assertion?